.github/workflows/packaging.yml

```diff
@@ -0,0 +1,26 @@
+name: Packaging
+
+on:
+  release:
+    types: [released]
+
+
+jobs:
+  winget:
+    runs-on: windows-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup winget-create
+        shell: powershell
+        run: |
+          iwr https://aka.ms/wingetcreate/latest -OutFile wingetcreate.exe
+      - name: Parse release version from github reference
+        run: Set-Item -PATH Env:\GITHUB_CLI_VERSION -Value ${{ github.ref }}.substring(1) 
```

Comment:
Niptick: is this a very long-winded way of doing `$env:GITHUB_CLI_VERSION = "..."`?
Any reason why it's not being done as a regular variable in the step where it's actually used?
```pwsh
$version = "v1.2.3".Substring(1)
wingetcreate.exe update --id GitHub.cli --url https://github.com/cli/cli/releases/download/v${version}/gh_${version}_windows_amd64.msi --version ${version} --out .\manifest\
```

Reply:
New to Powershell syntax, hence the complex way of doing this. Thanks for the update. I'll fix it.

Reply:
Updated the YAML with the suggestion